components/cli/internal/test/kubecli.go

```diff
@@ -177,16 +170,16 @@ func (kubeCli *MockKubeCli) GetServices(cellName string) (kubernetes.Services, e
 	return kubeCli.services[cellName], nil
 }
 
-func (kubeCli *MockKubeCli) GetCellLogsUserComponents(cellName string) (string, error) {
-	return kubeCli.cellLogs[cellName], nil
+func (kubeCli *MockKubeCli) GetCellLogsUserComponents(instanceName string, follow bool) (bool, error) {
+	return kubeCli.isInstanceAvailable, nil
```

Comment:
Better to check instance availability by mocking "GetInstanceAvailability".

Reply:
+1